pkg/controller/flink/ingress.go

```diff
@@ -58,6 +59,14 @@ func FetchJobManagerIngressCreateObj(app *flinkapp.FlinkApplication) *v1beta1.In
 			},
 		}},
 	}
+	if app.Spec.IngressTLSSecretName != "" {
+		ingressSpec.TLS = []v1beta1.IngressTLS{
+			{
+				Hosts:      []string{GetFlinkUIIngressURL(app.Name)},
```

Comment:
nit:  `Hosts:      []string{GetFlinkUIIngressURL(getIngressName(app)},`   to ensure the app name honors the BG deployment mode.